docs/datasource_api_userguide.md

```diff
@@ -242,73 +260,82 @@ CREATE TABLE tpch_test.TARGET_TABLE_CUSTOMER (
-| writeConcurrency            | -             | false    | maximum number of threads writing data to tikv, suggest `writeConcurrency` <= 8 * `number of TiKV instance` | 0       |
+| skipCommitSecondaryKey      | -             | false    | To skip commit secondary key                                                                                   | false   |
+| enableRegionSplit           | -             | false    | To split Region to avoid hot Region during insertion                                                        | true    |
+| regionSplitNum              | -             | false    | The Region split number defined by user during insertion                                                           | 0       |
+| replace                     | -             | false    | To define the behavior of append                                                                              | false   |
+| lockTTLSeconds              | -             | false    | tikv lock ttl. The write duration must be longer than `lockTTLSeconds`, otherwise write might fail because of the Garbage Collection (GC).              | 3600    |
+| writeConcurrency            | -             | false    | The maximum number of threads that write data to TiKV. It is recommended that `writeConcurrency` is smaller than or equal to 8 * `number of TiKV instance`. | 0       |
 
 ## TiDB Version and Configuration for Write
-TiDB's version should >= 3.0 and make sure that the following tidb's configs are correctly set.
+
+TiDB's version must be 3.0 or later.
+
+Make sure that the following TiDB configuration items are correctly set.
 
 ```
-# enable-table-lock is used to control table lock feature. The default value is false, indicating the table lock feature is disabled.
+# enable-table-lock is used to control the table lock feature. The default value is false, indicating that the table lock feature is disabled.
 enable-table-lock: true
 
-# delay-clean-table-lock is used to control the time (Milliseconds) of delay before unlock the table in the abnormal situation.
+# delay-clean-table-lock is used to control the time (milliseconds) of delay before unlocking the table in abnormal situations.
 delay-clean-table-lock: 60000
 
-# When create table, split a separated region for it. It is recommended to
-# turn off this option if there will be a large number of tables created.
+# When creating table, split a separated Region for it. It is recommended to
+# turn off this option if there is a large number of tables created.
 split-table: true
 ```
 
-If your TiDB's version is < 3.0 and want to have a try, you can set `spark.tispark.write.without_lock_table` to `true` to enable write, but no ACID is guaranteed.
+If your TiDB's version is earlier than 3.0 and you still want to have a try, set `spark.tispark.write.without_lock_table` to `true` to enable write, but ACID is **not** guaranteed.
```

Comment:
"you still want to have a try"... Try what? The meaning here is not so clear.

Reply:
Just delete `and you still want to have a try`...